Plugins/Flow.Launcher.Plugin.BrowserBookmark/Models/CustomBrowser.cs

```diff
@@ -1,45 +1,44 @@
-﻿namespace Flow.Launcher.Plugin.BrowserBookmark.Models
+﻿namespace Flow.Launcher.Plugin.BrowserBookmark.Models;
+
+public class CustomBrowser : BaseModel
 {
-    public class CustomBrowser : BaseModel
-    {
-        private string _name;
-        private string _dataDirectoryPath;
-        private BrowserType browserType = BrowserType.Chromium;
+    private string _name;
+    private string _dataDirectoryPath;
+    private BrowserType _browserType = BrowserType.Chromium;
```

Comment:
Add `_` to private property name to be consistent with other private property names

Reply:
I suppose you want to say private field (they are different in c#).

Reply:
Yeah, sorry, I meant field, not property.